src/box/lua/upgrade.lua

```diff
@@ -1243,6 +1257,7 @@ local handlers = {
     {version = mkversion(2, 7, 1), func = upgrade_to_2_7_1, auto = true},
     {version = mkversion(2, 9, 1), func = upgrade_to_2_9_1, auto = true},
     {version = mkversion(2, 10, 1), func = upgrade_to_2_10_1, auto = true},
+    {version = mkversion(2, 11, 0), func = upgrade_to_2_11_0, auto = true},
```

Comment:
AFAIU this should be back-ported to 2.10, right? If so, you should use (2, 10, 2) here.

Reply:
Asked @Mons, decided not to backport.